ci/docker/linux_opensuse_tumbleweed

```diff
@@ -0,0 +1,19 @@
+FROM opensuse/tumbleweed
+RUN \
+  zypper -n update && \
+  zypper -n install \
+  shadow util-linux util-linux-systemd systemd-devel \
+  git make gcc gcc-c++ perl-IPC-Run \
+  ccache clang meson ninja bison flex diff gettext-runtime \
+  llvm-devel icu libicu-devel liblz4-devel libopenssl-devel \
+  readline-devel pam-devel tcl-devel python3-devel \
+  libxml2-devel libxslt-devel libldapcpp-devel libuuid-devel \
+  krb5-devel krb5-client krb5-server krb5-plugin-kdb-ldap \
+  openldap2 openldap2-client
```

Comment:
For the debian container I now put these into one-package-per-line and grouped them into different sections. See https://github.com/anarazel/pg-vm-images/blob/containers/scripts/linux_debian_install_deps.sh
Perhaps worth copying?

Reply:
Done.